core/store/migrate/migrations/0113_add_last_keeper_id_to_upkeep_table.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE upkeep_registrations
+    ADD COLUMN last_keeper_index integer DEFAULT NULL;
```

Comment:
can we drop the positioning constant too?

Reply:
done